rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/190_knn_query-with-k-param.yml

```diff
@@ -0,0 +1,236 @@
+# test how knn query interact with other queries
+setup:
```

Comment:
Maybe we could also add a test where we have a `knn` query through a `standard` retriever?

Reply:
+1, added test in b5a189af78b9009f538bfc93662fa4ec8ae0e124